apps/web/app/(app)/environments/[environmentId]/integrations/google-sheets/components/GoogleSheetWrapper.tsx

```diff
@@ -55,22 +56,26 @@ export default function GoogleSheetWrapper({
             surveys={surveys}
             open={isModalOpen}
             setOpen={setModalOpen}
-            spreadsheets={spreadsheets}
             googleSheetIntegration={googleSheetIntegration}
             selectedIntegration={selectedIntegration}
           />
-          <Home
+          <ManageIntegration
             environment={environment}
             googleSheetIntegration={googleSheetIntegration}
             setOpenAddIntegrationModal={setModalOpen}
             setIsConnected={setIsConnected}
             setSelectedIntegration={setSelectedIntegration}
-            refreshSheet={refreshSheet}
           />
         </>
       ) : (
-        <Connect enabled={enabled} environmentId={environment.id} webAppUrl={webAppUrl} />
+        <ConnectIntegration
+          isEnabled={isEnabled}
+          integrationType={"googleSheets"}
+          handleAuthorization={handleGoogleAuthorization}
+          integrationLogoSrc={googleSheetLogo}
+          formbricksLogoSrc={formbricksLogo}
```

Comment:
why do we pass the formbricks logo to the component and not import it directly in the component if it's not changing?

Reply:
Importing svg/png in ui package, threw module not found error, but now created a component for formbricks logo ui/icons 😊